src/components/reusables/Panel.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from "react";
+import { GenericWedgeIcon } from "@nypl/dgx-svg-icons";
+
+export interface PanelOwnProps {
+  style?: string;
+  headerText: string;
+  body: JSX.Element;
+}
+
+export interface PanelState {
+  display: string;
+  icon: string;
+}
+
+export default class Panel extends React.Component<PanelOwnProps, PanelState> {
+  constructor() {
+    super();
+    this.renderHeader = this.renderHeader.bind(this);
+    this.toggle = this.toggle.bind(this);
+    this.state = { display: "collapse", icon: "down-icon" };
+  }
+
+  toggle() {
+    let display = this.state.display === "collapse" ? "" : "collapse";
+    let icon = this.state.icon === "down-icon" ? "up-icon" : "down-icon";
```

Comment:
Why not decide the icon based on state.display so you don't need to keep both in the state?